examples/DO_NOT_EDIT.md

```diff
@@ -0,0 +1 @@
+These example files are for the last stable Strimzi release. They should be updated only when a new Strimzi version is released, not as a part of regular PR. The editable files for the next release can be found in the [`packaging`](../packaging/) directory.
```

Comment:
we are not bringing this file because it has details about `packaging` folder which is related to Strimzi. You should remove it.

Reply:
Removed